appeals/web/src/server/appeals/appeal-details/interested-party-comments/interested-party-comments.mapper.js

```diff
@@ -1,6 +1,7 @@
 import { appealShortReference } from '#lib/appeals-formatter.js';
 import { dateISOStringToDisplayDate } from '#lib/dates.js';
 import { buildNotificationBanners } from '#lib/mappers/notification-banners.mapper.js';
+import { addressInputs } from '#lib/mappers/components/address.js';
```

Comment:
Lucy moved the ip mappers here in a recent PR

https://github.com/Planning-Inspectorate/appeals-back-office/tree/main/appeals/web/src/server/lib/mappers/page-components/interested-party-comments/view-and-review

Reply:
I think it makes sense to keep this code close to where it's used. Since this is only used by interested-party-comments routes, it's safe to just keep it here. We probably want to move it to lib if totally separate endpoints need to reuse the code.